templates/repo/actions/list.tmpl

```diff
@@ -13,12 +13,16 @@
 									{{svg "octicon-alert" 16 "text red"}}
 								</span>
 							{{end}}
+
+							{{if $.ActionsConfig.IsWorkflowDisabled .Entry.Name}}
+								<div class="ui red label">{{$.locale.Tr "disabled"}}</div>
+							{{end}}
 						</a>
 					{{end}}
 				</div>
 			</div>
 			<div class="twelve wide column content">
-				<div class="ui secondary filter stackable menu gt-je">
+				<div class="ui secondary filter stackable menu gt-je gt-df gt-ac">
```

Comment:
IIRC `flex` conflicts with `stackable menu`?

Have it been tested with small width screen?

Reply:
Hmm yes, does not look ideal

<img width="673" alt="image" src="https://github.com/go-gitea/gitea/assets/115237/696bb5ab-93df-43e1-aa13-d386473b898a">

Reply:
Fixed, see below.